Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
whenever people overwrite this method they could simply get the request from the `request_stack` though :wink: